### PR TITLE
Renaming all ID to Id

### PR DESCRIPTION
**Pull request by alshaw**
## Description

Renamed all ID to Id

## Related Issue

https://jira.corp.adobe.com/browse/CORE-31918

## Motivation and Context

To be consistent with the use of Id

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All tests pass and I've made any necessary test changes.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: sandbox/src/OrgTwo.js [hunk 1]
```diff
@@ -4,20 +4,20 @@ export default function OrgTwo() {
     return (
         <div>
         <h2>Organization Two</h2>
-        <h3>This view is managed by a partnering company that owns a different Org ID.</h3>
-        <p>For that reason, we have created a second instance of Alloy, and configurered it using the Org and Property IDs of Organization Two.</p>
+        <h3>This view is managed by a partnering company that owns a different Org Id.</h3>
+        <p>For that reason, we have created a second instance of Alloy, and configurered it using the Org and Property Ids of Organization Two.</p>
```
Comment (Aaronius) [inline sandbox/src/OrgTwo.js]: In non-programatic contexts, like here where we're just communicating in English, we should probably keep ID all-caps.

**Comment by jfkhoury** [comment]
@alshaw Wanna make the same change in the Mock gateway and submit your first PR there?

https://git.corp.adobe.com/Activation/edge-gateway-mock

**Comment by Aaronius** [comment]
@alshaw, I noticed this line needs to be updated as well: https://github.com/adobe/alloy/blob/1837f627ee3bab2a358e337255068703bb546321/sandbox/public/index.html#L39

**Comment by jfkhoury** [comment]
Ticket on the Konductor side to rename `ID` to `Id`: https://jira.corp.adobe.com/browse/EXEG-104

**Pull request merged**